SI/Sem.03/Pract.03/Task05.cpp

```diff
@@ -0,0 +1,26 @@
+#include <iostream>;
+using namespace std;
+
+int main()
+{
+    int targetNumber;
+    int counter = 0;
+    int guessNumber;
+
+    cout << "Please enter whole number from 0 to 100: ";
+    cin >> targetNumber;
+
+    do
+    {
+        cout << "Enter your guess: ";
+
+        cin >> guessNumber;
+        counter++;
+
```

Comment:
if (guessNumber < targetNumber)
				cout << "The number you are looking for is greater than your guess!\n";
 else if (guessNumber > targetNumber)
				cout << "The number you are looking for is less than your guess!\n";